examples/send-zerocopy.c

```diff
@@ -104,13 +110,17 @@ static int do_setup_tx(int domain, int type, int protocol)
 	int fd;
 
 	fd = socket(domain, type, protocol);
-	if (fd == -1)
-		error(1, errno, "socket t");
+	if (fd == -1) {
+		fprintf(stderr, "socket t: %s\n", strerror(errno));
```

Comment:
I tried to print a similar message here by adding a ```: %s``` and pass ```strerror(int)``` to ```fprintf``` where there was a non-zero error number as a parameter before. If this is not needed and/or wanted I would remove it for all ```fprintf``` calls.